src/system/SystemPacketBuffer.h

```diff
@@ -230,3 +243,86 @@ inline uint16_t PacketBuffer::AllocSize() const
 
 } // namespace System
 } // namespace chip
+
```

Comment:
Is there any strong reason to not create `SystemPacketBufferHandle.h` and `SystemPacketBufferHandle.cpp`, move most of the implementation into the .cpp file and have `SystemPacketBuffer.h` include `SystemPacketBufferHandle.h`?

Reply:
There was a desire expressed to have PacketBufferHandle largely inline.